client/src/partials/patient_invoice/registry/modalCreditNote.html

```diff
@@ -0,0 +1,70 @@
+<form  name="ModalForm" ng-submit="ModalCtrl.submit(ModalForm)" bh-form-defaults novalidate>
+  <div class="modal-header">
+    <div class="bhima-title">
```

Comment:
In the future, do not use the class `bhima-title` in the modal headers.  @sfount has fixed modal header styling so that it displays a nice `headercrumb` without the `bhima-title` class.